cli-commands/aeproject-epoch/config.json

```diff
@@ -0,0 +1,53 @@
+{
+	"config": {
+		"host": "http://localhost:3001/",
+		"internalHost": "http://localhost:3001/internal/",
+		"keyPair": {
+			"secretKey": "bb9f0b01c8c9553cfbaf7ef81a50f977b1326801ebf7294d1c2cbccdedf27476e9bbf604e611b5460a3b3999e9771b6f60417d73ce7c5519e12f7e127a1225ca",
+			"publicKey": "ak_2mwRmUeYmfuW93ti9HMSUJzCk1EYcQEfikVSzgo6k2VghsWhgU"
+		},
+		"nonce": 1
```

Comment:
We don't need any more nonce?

Reply:
Actually, we need it. We can investigate why it is not working because of the nonce in a separate task.